modules/API/pages/gsql-endpoints.adoc

```diff
@@ -4162,3 +4162,2356 @@ Response::
 ====
 
 '''
+
+=== get loading job names
+
+`GET /gsql/v1/loading-jobs?graph=[graphName]`
+
+This endpoint is used to get all loading job names of a graph.
+
+==== Example
+
+[,tabs]
+====
+Request::
++
+--
+[source,bash]
+----
+curl -H 'Content-Type: application/json' -X GET 'http://localhost:14240/gsql/v1/loading-jobs?graph=g' 
+----
+--
+Response::
++
+--
+[source.wrap,console]
+----
+{"error":false,"message":"","jobNames":["incidents_fraud_report_company_csv"]}
+----
+--
+====
+===== Parameters:
+
+//[width="100%",cols="25%,25%,50%",options="header",]
+[%autowidth]
+|===
+| Name | Required | Description
+
+| `graph`
+| yes
+| the loading job names of which graph
+|===
+
+'''
+
+=== create/update a loading job
+
+`POST/PUT /gsql/v1/loading-jobs?graph=[graphName]`
+
+This endpoint is used to create/update a new/existed loading job.
+
+==== Example
+
+[,tabs]
+====
+Request::
++
+--
+[source,bash]
+----
+curl -H 'Content-Type: text/plain' -X POST/PUT 'http://localhost:14240/gsql/v1/loading-jobs?graph=g' -d '[loading job statement]'

```

Comment:
Change `[loading job statement]` to `<loading job statement>`, "[]" is used to represent array in json.